src/nu/xom/Builder.java

```diff
@@ -188,64 +188,67 @@ public Builder(boolean validate, NodeFactory factory) {
         "org.xmlpull.v1.sax2.Driver" // android
     };
 
-    
+
     static XMLReader findParser(boolean validate) {
-        
+
         // first look for Xerces; we only trust Xerces if
         // we set it up; and we need to configure it specially
         // so we can't load it with the XMLReaderFactory
-        XMLReader parser; 
+        XMLReader parser;
         try {
             parser = new XML1_0Parser();
             setupParser(parser, validate);
             return parser;
-        } 
+        }
         catch (SAXException ex) {
             // look for next one
         }
         catch (NoClassDefFoundError err) {
             // Xerces is not available; look for next one
-        } 
+        }
 
         try {
             parser = (XMLReader) Class.forName(
               "nu.xom.JDK15XML1_0Parser").newInstance();
             setupParser(parser, validate);
             return parser;
-        } 
+        }
         catch (SAXException ex) {
             // look for next one
         }
         catch (InstantiationException ex) {
             // look for next one
-        } 
+        }
         catch (ClassNotFoundException ex) {
             // look for next one
         }
         catch (IllegalAccessException ex) {
             // look for next one
         }
         catch (NoClassDefFoundError err) {
-            // Xerces is not available; look for next one
-        } 
-        
+          // Xerces is not available; look for next one
+      }
```

Comment:
The irony, it burns! Despite all the whitespace changes the indentation isn't the same here.

Reply:
Seems like my VSCode setup is too smart for me. 🙈  I should have used vim...

Sorry for the whitespaces... the change is trivial anyway, so I think I won't bother you with another PR...